spec/src/modules/search.js

```diff
@@ -317,15 +320,17 @@ describe(`ConstructorIO - Search${bundledDescriptionSuffix}`, () => {
         fetch: fetchSpy,
       });
 
-      search.getSearchResults(query, { section, hiddenFacets }, {}).then((res) => {
+      search.getSearchResults('item1', { section, hiddenFacets }, {}).then((res) => {
         const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
 
+        const facetWithNameBrand = res.response.facets.find(facet => facet.name === 'Brand');
+
         expect(res).to.have.property('request').to.be.an('object');
         expect(res).to.have.property('response').to.be.an('object');
         expect(res).to.have.property('result_id').to.be.an('string');
         expect(res.request.fmt_options.hidden_facets).to.eql(hiddenFacets);
         expect(requestedUrlParams.fmt_options).to.have.property('hidden_facets').to.eql(hiddenFacets);
-        expect(res.response.facets[0]).to.have.property('name').to.eql('Brand');
+        expect(facetWithNameBrand).to.be.not.null;
```

Comment:
Same for here, this will always be true at the moment.

Reply:
Got it, added more precise tests instead.